accounts/models.py

```diff
@@ -10,7 +10,7 @@
 
 class Account(AbstractBaseUser, PermissionsMixin):
     email = models.EmailField(max_length=255, unique=True)
-    username = models.CharField(max_length=255, unique=True)
+    username = models.CharField(max_length=255, unique=True, verbose_name='station name')
```

Comment:
follow this pattern: double quote strings for strings that user will see (like "station name") and single quote strings for string that user will not see.